core/src/i18n/en.yaml

```
@@ -156,6 +158,7 @@ btp-service-bindings:
     secret: The name of the secret where the credentials are stored, defaults to the ServiceBinding name if not specified.
     secret-not-found: This Secret doesn't exist.
 catalog:
+  description: <0>Service Catalog</0> provides an easy way to access services that the Service Brokers manage and use them in an application.
```

ambiguous.

If it's the human who uses the services, I suggest
```suggestion
  description: <0>Service Catalog</0> provides an easy way to access services that the Service Brokers manage, and to use them in an application.
```
If it's the Service Brokers who use the services, I suggest

```suggestion
  description: <0>Service Catalog</0> provides an easy way to access services that the Service Brokers manage and use in an application.
```